randovania/gui/multiplayer_session_window.py

```diff
@@ -787,6 +803,10 @@ async def import_permalink(self):
             return
 
         permalink = dialog.get_permalink_from_field()
+        has_unsupported_games = self._check_for_unsupported_games(permalink.parameters.presets)
+        if has_unsupported_games is not None:
+            return await has_unsupported_games
```

Comment:
i would have the function return a set of unsupported games, and have the `if len(unsupported_games) != 0` part in another function that can be called with `show_dialog_for_unsupported_games(unsupported_games)`. reads nicer IMO.

Reply:
Changed it. Is this better?